Guess game/Program.cs

```diff
@@ -1,89 +1,126 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Data;
-using System.Linq;
-using System.Runtime.InteropServices;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Guess_game
 {
     class Program
     {
         private static Dictionary<string, string> Riddles = new Dictionary<string, string>
         {
-            {"zalia zole bet ne zole su uodega bet ne pele?" , "agurkas"},
-            { "be ranku be koju duris atidaro?", "vejas"},
-            { "I pakalne letai i kalna greitai?", "snarglys"}
+            {"Žalia žole, bet ne žole su uodega, bet ne pelė?" , "agurkas"},
+            { "Be rankų, be kojų duris atidaro?", "vejas"},
+            { "Į pakalnę lėtai, į kalną greitai?", "snarglys"}
         };
 
+        private static int RiddlesPass = Riddles.Count;
```

Comment:
If you use field value only in one method, when don't introduce such field because it has no value to be a field, you can write this logic  `Riddles.Count` in console writeline directly: Console.WriteLine($"* jūs turėsite įminti {Riddles.Count} mįsles");`. Fields is only useful if you use it in multiple methods